server/modules/aws-notifier.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const AWS = require('aws-sdk')
+
+const AWSNotifier = function () {
+  let SIMPLE_NOTIFICATION_SERVICE;
+
+  (function (obj) {
+    AWS.config.loadFromPath('./configs/aws-config.json')
+    SIMPLE_NOTIFICATION_SERVICE = new AWS.SNS()
+  })(this)
+
+  this.publish = function (note, callback) {
+    let params = {
+      Subject: note.subject,
+      Message: note.message,
+      TopicArn: note.TopicArn
+    }
+    SIMPLE_NOTIFICATION_SERVICE.publish(params, callback)
+  }
+  this.createTopic = function (topicName, callback) {
```

Comment:
why we want to create SNS topic from the code. we expect it has to be created as CloudFormation Resource from the serverless.yml file

Reply:
We should not create a Topic from the code